mono/mini/aot-runtime.c

```diff
@@ -3114,7 +3114,7 @@ decode_exception_debug_info (MonoAotModule *amodule, MonoDomain *domain,
 			}
 		}
 
-		flags |= JIT_INFO_HAS_UNWIND_INFO;
+		flags = (MonoJitInfoFlags)(flags | JIT_INFO_HAS_UNWIND_INFO);
```

Comment:
still needed after `G_ENUM_FUNCTIONS (MonoJitInfoFlags)` ?

Reply:
Good catch. Unlikely still needed.

Reply:
https://github.com/mono/mono/issues/10319 (can undo this line later)